index.js

```diff
@@ -641,6 +641,7 @@ class GithubScm extends Scm {
         const type = payloadHeaders['x-github-event'];
         const hookId = payloadHeaders['x-github-delivery'];
         const checkoutUrl = hoek.reach(webhookPayload, 'repository.ssh_url');
+        const scmContexts = this._getScmContexts();
```

Comment:
In [this](https://github.com/screwdriver-cd/scm-base/pull/43/commits/7abae55ccd0648807675f74f7bbe82fd9b8fac0c) commit, the type of the return value of _getScmContexts() was changed from Promise to Array. So I think that `scmContexts` can be acquired in this way.